pyproject.toml

```diff
@@ -6,7 +6,7 @@ build-backend = "hatchling.build"
 name = "codecarbon"
 dynamic = ["version"]
 readme = "README.md"
-license = ""
+license = {file = "LICENSE"}
```

Comment:
It is optional because already in `classifiers`, maybe it's better to remove this line ?